oak-lucene/pom.xml

```diff
@@ -358,6 +358,13 @@
       <version>${jackrabbit.version}</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.apache.jackrabbit</groupId>
+      <artifactId>oak-search</artifactId>
+      <version>${project.version}</version>
+      <type>test-jar</type>
```

Comment:
Is this needed ? Shouldn't scope=test be sufficient ?

Reply:
each module in oak build src and test jar. Here oak-search's test jar is required  as abstract tests are written in oak-search-test jar and hence type test-jar . In this pom itself oak-search(src) dependency is also included above.